config-builder/src/main/java/com/ca/apim/gateway/cagatewayconfig/beans/TrustedCert.java

```diff
@@ -202,6 +226,29 @@ public void preWrite(File configFolder, DocumentFileUtils documentFileUtils) {
         this.certificateData = null;
     }
 
+    @Override
+    public AnnotatedEntity getAnnotatedEntity() {
+        if (annotatedEntity == null && annotations != null) {
+            annotatedEntity = createAnnotatedEntity();
+        }
+        return annotatedEntity;
+    }
+
+    @VisibleForTesting
+    public void setAnnotatedEntity(AnnotatedEntity<Encass> annotatedEntity) {
+        this.annotatedEntity = annotatedEntity;
+    }
+
+    @Override
+    public String getType() {
+        return EntityTypes.TRUSTED_CERT_TYPE;
+    }
+
+    @Override
+    public String getShortenedType() {
```

Comment:
This was removed I suppose. Can you pick up those changes?

Reply:
@nikhilahuja , yes, will do

Reply:
done